src/storage/FileResourceMapper.ts

```diff
@@ -0,0 +1,48 @@
+import { types } from 'mime-types';
+import { ResourceIdentifier } from '../ldp/representation/ResourceIdentifier';
+import { NotFoundHttpError } from '../util/errors/NotFoundHttpError';
+import { FileResourceStore } from './FileResourceStore';
+import { ResourceMapper } from './ResourceMapper';
+
+export class FileResourceMapper implements ResourceMapper {
+  private readonly fileStore: FileResourceStore;
+  private readonly types: Record<string, any>;
+
+  public constructor(fileStore: FileResourceStore, overrideTypes = { acl: 'text/turtle', metadata: 'text/turtle' }) {
+    this.fileStore = fileStore;
```

Comment:
Let's indeed not pass the whole store, but just the settings we need. Store should depend on a mapper, but not the other way round. Plus, it now seems that the store itself should not need to know a couple of those parameters anymore then, such as `baseRequestURI`.